.gitignore

```diff
@@ -8,9 +8,6 @@ local.config
 *.o
 *.mod
 
-# Ignore all PNGs
-*.png
-
```

Comment:
Just noting that we discussed this.  Ignoring PNG files causes trouble for the documentation so we don't want to include these in the `.gitignore.